### PR TITLE
Add Buefy Logo

### PR DESCRIPTION
**Pull request by service-paradis**
buefy

**Issue:** #3420
**Alexa rank:** 73,851
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I followed the contributing guidelines in order to make this PR. I used https://raw.githubusercontent.com/buefy/buefy/dev/static/img/icons/safari-pinned-tab.svg as a starting point. It was already a monochromatic vector. I used Inkscape in order to meet the requirements. Finally, I used https://jakearchibald.github.io/svgomg/ to optimize the SVG.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -940,6 +940,11 @@
             "hex": "1A86FD",
             "source": "https://buddy.works/about"
         },
+        {
+            "title": "Buefy",
+            "hex": "7957D5",
+            "source": "https://github.com/buefy/buefy"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:946]: Following [the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), could you update this to:

```suggestion
            "source": "https://github.com/buefy/buefy/blob/a9a724efca0b531e6a64ab734889b00bf4507a9d/static/img/icons/safari-pinned-tab.svg"
```

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
SVG looks good [when compared to a raster version of the logo](https://user-images.githubusercontent.com/3742559/93365812-ee76ef00-f852-11ea-94dd-22f183257d33.png) so I definitely accept that 👍 The rest of the PR looks good except for one minor thing (see below). Once that is changed this is ready to merge 😄

**Comment by service-paradis** [comment]
Thanks for your comment. I just pushed a fix :)

**Review by ericcornelissen — APPROVED** [review]
Thanks for the quick fix @service-paradis, your contribution is much appreciated! :confetti_ball:

**Pull request merged**